src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -396,7 +396,15 @@
     <span class="ps-switch">
         {% for choice in choices %}
             {% set inputId = id ~'_' ~ choice.value %}
-            <input id="{{inputId}}" {{ block('attributes') }} name="{{ full_name }}" value="{{ choice.value }}" {%- if choice is selectedchoice(value) -%}checked="" {%- endif -%} {%- if disabled -%}disabled="" {%- endif -%} type="radio" aria-label="{{ choice.label|trans({}, choice_translation_domain) }}">
+            <input id="{{inputId}}"
+                {{ block('attributes') }}
+                name="{{ full_name }}"
+                value="{{ choice.value }}"
+                {% if choice is selectedchoice(value) %}checked=""{% endif %}
+                {% if disabled %}disabled=""{% endif %}
```

Comment:
Maybe one of the following two:
```suggestion
                {% if choice is selectedchoice(value) %}checked="checked"{% endif %}
                {% if disabled %}disabled="disabled"{% endif %}
```
```suggestion
                {% if choice is selectedchoice(value) %}checked{% endif %}
                {% if disabled %}disabled{% endif %}
```

Reply:
I'm not sure because disabled alone means `disabled="disabled"`
I'm afraid of side effect, (and it's not related to the PR)

Reply:
disabled="" === disabled="disabled" === disabled="true" === disabled="false"

Every values are the same, same for checked, you can just leave `checked` and `disabled` I think

Reply:
But well it's not related anyway :)